addons/base_import/models/base_import.py

```diff
@@ -859,6 +859,9 @@ def _import_image_by_url(self, url, session, field, line_number):
             if response.headers.get('Content-Length') and int(response.headers['Content-Length']) > maxsize:
                 raise ValueError(_("File size exceeds configured maximum (%s bytes)", maxsize))
 
+            SUPPORTED_IMAGE_MIMETYPES = ['image/jpe', 'image/jpeg', 'image/jpg', 'image/gif', 'image/png', 'image/svg+xml']
+            if response.headers.get('Content-Type', '') not in SUPPORTED_IMAGE_MIMETYPES:
+                return False
```

Comment:
```suggestion
                raise ValueError(_("Invalid file type"))
```
I think raising a valueError, instead of returning false could be a better idea
Maybe not exactly with this sentance, but something like that (with the translated term in .pot)